rules/execute-command.js

```diff
@@ -132,33 +131,39 @@ async function runCommand(command, args = []) {
   }
 }
 
-function smuggleLockfile(rootDir) {
+function smuggleYarnProjectFiles(rootDir) {
   // Support non-hermetic yarn.lock/.pnp.cjs/.pnp.loader.mjs smuggling into sandbox
   // This allows a repo to use yarn plugins to implement custom change detection
   // to avoid invalidating top-level cache in cases where yarn.lock is touched but only affects certain projects
   // See: Lockfile delegation section in README
 
-  try {
-    const realRoot = dirname(realpath(`${rootDir}/package.json`)); // FIXME: technically a target may not depend on this file, so it's not actually guaranteed to exist, even though it usually does
+  // @see {@link https://github.com/yarnpkg/berry/blob/07cf3531002a3f25fd6309e05d0cf1b233c59cd4/packages/yarnpkg-fslib/sources/path.ts#L21-L35}
+  const yarnProjectFiles = [
+    'yarn.lock',
+    '.pnp.data.json',
+    '.pnp.loader.mjs',
+    '.pnp.cjs',
+    '.yarnrc.yml',
+    '.yarnrc',
```

Comment:
yarnrc rarely ever changes, might not need to smuggle

Reply:
Yeah, I'm actually not sure what this file is used for nowadays. And if its `.yarnrc.yml` YAML counterpart completely supersedes it?

I'll ditch it